mentoring/templates/html/dashboard_report.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
this won't be right if/when your block is i18n'ed. 

that said I'm not sure how we tackle this in edx-platform, or if we have a plan for xblocks. maybe just something to keep in the back of our minds.